cylc/flow/task_events_mgr.py

```diff
@@ -807,10 +807,13 @@ def _process_message_check(
                 )
             return False
 
-        LOG.log(
-            LOG_LEVELS.get(severity, INFO),
-            f"[{itask}] {flag}{message}{timestamp}"
-        )
+        severity_lvl = LOG_LEVELS.get(severity, DEBUG)
```

Comment:
I think Hillary made a good case for the `INFO` default with custom outputs as I don't think these would get logged elsewhere.

Reply:
Custom outputs are still logged at `INFO` by default